README.md

```diff
@@ -126,12 +127,14 @@ The example below is a simplified version of fio benchmark requirements file:
 ```
 
 The order of precedence for overriding params is the following:
-    1. `essentials`: always use (override params defined elsewhere).
-    2. param sets defined in the input file: apply params overriding defaults and
-       `presets`.
-    3. `presets`: override all default params with the params in the `presets`
-       array.
-    4. `defaults`: only use if not defined anywhere else.
+    1. `sets`: param sets defined in the input file override all params
+       defined elsewhere.
+    2. `global-options`: params included override `presets`.
+    3. `named presets`: override all `defaults` and `essentials` params.
+    4. `essentials`: only use params if not defined anywhere else. If the
+       param is already defined, ignore it.
```

Comment:
This doesn't sound right.  The essentials should always override anything else, unless we add some kind `--force` option or something similar.

Reply:
ok I can change it again.... so for now I'll make it override any existing param